### PR TITLE
Change slide for ReddyNeeta

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random meme for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [x] @jimmieego
- [ ] @intheswim
- [ ] @epitrochoider
- [ ] @githubteacher
- [ ] @skrishn
- [ ] @TheMalaka
- [ ] @weiy0415
- [ ] @shub8703
- [ ] @AshPengelly
- [ ] @mpurohit1
- [ ] @reddyneeta
- [ ] @Lyuye
- [ ] @Lyuye
- [ ] @ak-esri

### PR DESCRIPTION
**Pull request by reddyneeta**
Notify others for attention.

Ask leading questions.
-Questions here
-Another question

Refer to other issues
related to #2 
closes #2

### CODE REVIEW

**Review by ak-esri — COMMENTED** [review]
Don't you think Hansolo will be better helper??

**Review by ak-esri — CHANGES_REQUESTED** [review]
Obi-wan is so frail ... he can't help ... he needs help himself !!

**Comment by ak-esri** [comment]
@mpurohit1 what do you think ... is this creative enough??

File: _posts/0000-01-12-reddyneeta.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 layout: slide
-title: "Welcome to our slide deck!"
+title: "ReddyNeeta's Slide"
 ---
 
 ![leia](https://cloud.githubusercontent.com/assets/16547949/25400918/17c5d2e4-29c2-11e7-92ef-79bacb424ef4.jpg)
 {: .center}
 
-Caption here
+Help me Obi-Wan Kenobi, you're my only hope...
```
Comment (ak-esri) [inline _posts/0000-01-12-reddyneeta.md]: ReddyNeeta is not ready

**Review by ak-esri — APPROVED** [review]
🥇 good job !

File: _posts/0000-01-12-reddyneeta.md (hunk 1 above)
Comment (reddyneeta) [reply _posts/0000-01-12-reddyneeta.md]: I am ever ready!

**Review by ak-esri — APPROVED** [review]
yayy

**Pull request merged**